apps/api/src/app/workflows-v2/shared/schemas/in-app-control.schema.ts

```diff
@@ -9,33 +9,36 @@ const redirectSchema = {
     url: {
       type: 'string',
       pattern: ABSOLUTE_AND_RELATIVE_URL_REGEX,
+      default: '',
```

Comment:
Added the default values to the `dataSchema`, these are used to generate the default values for the form. The default values are required for all visible text fields by the `react-hook-form`, [ref](https://react-hook-form.com/docs/useform#defaultValues).